lib-ng/mapper/beuckman.js

```diff
@@ -12,4 +12,17 @@ const map = beaconRaw => ({
   proximity: beaconRaw.proximity,
 });
 
-export default { map };
+const region = bleRegion => {
+  const beaconRaw = bleRegion.identifier.split('|');
+
+  return {
+    UUID: beaconRaw[1].toUpperCase(),
+    major: parseInt(beaconRaw[2], 10),
+    minor: parseInt(beaconRaw[3], 10),
+    rssi: -1,
```

Comment:
Why are the `rssi`, `accuracy` and `proximity` needed here? You're defining a region... not a result.